base/shell/explorer/traywnd.cpp

```diff
@@ -1788,6 +1592,7 @@ class CTrayWindow :
         SetWindowPos(NULL, 0, 0, 0, 0, SWP_NOZORDER | SWP_NOACTIVATE);
         ResizeWorkArea();
         ApplyClipping(TRUE);
+        RedrawWindow(NULL, NULL, RDW_ERASE | RDW_FRAME | RDW_INTERNALPAINT | RDW_INVALIDATE | RDW_ALLCHILDREN); // | RDW_ERASENOW | RDW_UPDATENOW);
```

Comment:
Adding the last two flags generate glitches, or are unnecessary?

Reply:
...is that a question or a statement?

Reply:
It's a question (I've fixed it to look like a real question)

Reply:
@HBelusca They're not needed - I think I commented them out while trying to figure out which flags I needed, and forgot to remove the comment after. I'll uh...get rid of that comment now.